### PR TITLE
not all nested keys were added

### LINKED ISSUE #207: [config-transformer] not all things are transformed
first of all thanks for the package. It is really helpful

to see what was not changed I'll provide original and transformed files

```yaml
foo:
    path: /asd/das/{foo}
    controller: Asd\Das
    methods: GET
    requirements:
      foo: '[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}'
```

transformed to

```php
<?php

declare(strict_types=1);

use Asd\Das;
use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;

return static function (RoutingConfigurator $routingConfigurator): void {
    $routingConfigurator->add('foo', '/asd/das/{foo}')
        ->controller(Das::class);
};

```

see that `requirements` and `methods` is missing

### PR DESCRIPTION
**Pull request by vladyslavstartsev**
closes #207

### CODE REVIEW

**Comment by vladyslavstartsev** [comment]
there are fails because of something with `token`

File: packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/full_import.yaml [hunk 1]
```diff
@@ -20,7 +20,14 @@ api_platform:
...
     stateless: false
------
+
+foo:
+    path: /asd/das/{foo}
+    methods: 'GET|POST'
+    requirements:
+        foo: '[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}'
+    controller: Asd\Das
+    -----
```
Comment (TomasVotruba) [inline packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/full_import.yaml:30]: Could you decouple this one to a new test fixture file? Just so we have less scope to maintain later.

Thank you

File: packages/php-config-printer/src/RoutingCaseConverter/PathRoutingCaseConverter.php [hunk 2]
```diff
@@ -53,7 +58,14 @@ public function convertToMethodCall(string $key, $values): Expression
                 continue;
             }
 
-            $args = $this->argsNodeFactory->createFromValues([$values[$nestedKey]]);
+            $nestedValues = $values[$nestedKey];
+
+            // Transform methods as string GET|HEAD to array
+            if ($nestedKey === self::METHODS && is_string($nestedValues)) {
+                $nestedValues = explode('|', $nestedValues);
+            }
```
Comment (TomasVotruba) [inline packages/php-config-printer/src/RoutingCaseConverter/PathRoutingCaseConverter.php:66]: :clap:

File: packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/full_import.yaml (hunk 1 above)
Comment (vladyslavstartsev) [reply packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/full_import.yaml]: I've moved all the changes to appropriate place - `routes.yaml`

File: packages/php-config-printer/src/RoutingCaseConverter/PathRoutingCaseConverter.php (hunk 2 above)
Comment (vladyslavstartsev) [reply packages/php-config-printer/src/RoutingCaseConverter/PathRoutingCaseConverter.php:66]: Anything wrong with this one?

Comment (TomasVotruba) [reply packages/php-config-printer/src/RoutingCaseConverter/PathRoutingCaseConverter.php:66]: No, it's "a clapping" like :+1: 
All good!

File: packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/routes.yaml [hunk 3]
```diff
@@ -4,6 +4,9 @@ research_thank_you:
     controller: 'Symfony\Bundle\FrameworkBundle\Controller\TemplateController'
     defaults:
         template: 'research/thank-you.twig'
+    methods: 'GET|POST'
+    requirements:
+        foo: '[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}'
```
Comment (TomasVotruba) [inline packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/routes.yaml:9]: Could you split this fixture into 2 files, it's easier to debug later?
Thanks

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**